src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -382,24 +381,23 @@
          token-decimal (when token (:decimals token))
          token-id (utils/format-token-id token collectible)
          to-token-id ""
-         network-preferences (if token [] [(get-in collectible [:id :contract-id :chain-id])])
          gas-rates constants/gas-rate-medium
          to-hex (fn [v] (send-utils/amount-in-hex v (if token token-decimal 0)))
          amount-in (to-hex amount)
+         amount-out (to-hex amount-out)
          from-address wallet-address
-         disabled-from-chain-ids disabled-from-chain-ids
          disabled-to-chain-ids (if (= transaction-type :tx/bridge)
                                  (filter #(not= % bridge-to-chain-id) network-chain-ids)
                                  (filter (fn [chain-id]
                                            (not (some #(= chain-id %)
                                                       receiver-networks)))
                                          network-chain-ids))
          from-locked-amount (update-vals from-locked-amounts to-hex)
-         transaction-type-param (case transaction-type
-                                  :tx/collectible-erc-721  constants/send-type-erc-721-transfer
-                                  :tx/collectible-erc-1155 constants/send-type-erc-1155-transfer
-                                  :tx/bridge               constants/send-type-bridge
-                                  constants/send-type-transfer)
+         send-type (case transaction-type
```

Comment:
I know this is status-go domain, but I am wondering why this parameter is called SendType when some values from this enum specify types of operations that doesn't involve sending (ENSRegister, StickersBuy, Swap, Bridge, etc). IMO, TransactionType is still a better name.